### PR TITLE
docs: add appProtocol for protocol support in k8s

### PR DESCRIPTION
**Pull request by tharun208**
Signed-off-by: Tharun <rajendrantharun@live.com>

### CODE REVIEW

File: docs/docs/1.0.7/policies/protocol-support-in-kuma.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ By doing this,
 
 :::: tabs :options="{ useUrlFragment: false }"
 ::: tab "Kubernetes"
-On `Kubernetes`, to give Kuma a hint that your service supports `HTTP` protocol, you need to add a `<port>.service.kuma.io/protocol` annotation to the `k8s` `Service` object.
+On `Kubernetes`, to give Kuma a hint that your service supports `HTTP` protocol, you need to add a `<port>.service.kuma.io/protocol` annotation to the `k8s` `Service` object or use `appProtocol` of the `k8s` `Service` object.
```
Comment (jakubdyszkiewicz) [inline docs/docs/1.0.7/policies/protocol-support-in-kuma.md:17]: please provide tabs `Kubernetes < 1.18`, `Kubernetes 1.18+`. I think mixing both approaches might be confusing for users

**Comment by tharun208** [comment]
@jakubdyszkiewicz is it good to get merged?

**Comment by nickolaev** [comment]
Oh this will need lots of forward porting , up till 1.1.5. @Bradamant3 shall we just make it 1.1.x (all versions?)

**Comment by Bradamant3** [comment]
@nickolaev @tharun208 I notice that the PR is still against the 1.0.7 directory. Do we want to take changes against this one all the way up to 1.1.15? (Sadly, VuePress doesn't support a 1.1.x directory naming scheme -- another reason I'm trying to get help migrating to Jekyll.)

And the PR is against the `next` branch -- can we retarget to `master` please?

**Comment by Bradamant3** [comment]
Closing so I can clean up broken `next` branch. Not sure whether we should reopen but if anyone is willing to tackle it, feel free.

**Pull request closed without merging**